.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,66 +1,30 @@
-<!--
-  ☝️How to write a good PR title:
-  - Prefix it with [ComponentName] (if applicable), for example: [Button]
-  - Start with a verb, for example: Add, Delete, Improve, Fix…
-  - Give as much context as necessary and as little as possible
-  - Prefix it with [WIP] while it’s a work in progress
--->
+<!-- PR title: [ComponentName] *Verb*ing a ... -->
 
 ### WHY are these changes introduced?
 
-Fixes #0000 <!-- link to issue if one exists -->
-
-<!--
-  Context about the problem that’s being addressed.
--->
+Fixes #0000
 
 ### WHAT is this pull request doing?
 
-<!--
-  Summary of the changes committed.
+Before / after screenshots and gifs are appreciated for UI changes
 
-  Before / after screenshots are appreciated for UI changes. Make sure to include alt text that describes the screenshot.
```

Comment:
I definitely understand wanting to shorten this, but I'd like to keep the guidance around putting animated gifs in a `<details>` block, and providing `alt` text. It helps make our PRs more accessible to more people, and it's not something that folks currently do by default.

Reply:
I get the alt text, but why does putting gifs in a details block make it more accessible?

Reply:
> I get the alt text, but why does putting gifs in a details block make it more accessible?

Gifs autoplay, which can be really disorienting and makes the content around it hard to read/focus on. It also bloats the content of the description, so it's good practice in general to collapse visuals so the main/important context of the PR isn't lost in a see of images, moving or not.